common/scala/src/main/scala/whisk/core/WhiskConfig.scala

```diff
@@ -216,6 +216,8 @@ object WhiskConfig extends Logging {
 
     val invokerContainerNetwork = "invoker.container.network"
     val invokerContainerPolicy = "invoker.container.policy"
+    val invokerNumCore = "invoker.numcore"
+    val invokerCoreShare = "invoker.coreshare"
```

Comment:
I think these are not enough, you'll need a "getter" above in that class. Please augment that to the PR actually using these values. Missed it before merging.